### PR TITLE
Maven DependencyGraphBuilder Mojo Skeleton

### PR DESCRIPTION
**Pull request by ian-lavallee**

### CODE REVIEW

File: verboseTree-maven-plugin/src/main/java/HelloWorldMojo.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 Google LLC.
```
Comment (elharo) [inline verboseTree-maven-plugin/src/main/java/HelloWorldMojo.java:2]: 2020

File: verboseTree-maven-plugin/src/test/java/HelloWorldMojoTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 Google LLC.
```
Comment (elharo) [inline verboseTree-maven-plugin/src/test/java/HelloWorldMojoTest.java:2]: 2020

**Comment by suztomo** [comment]
google-java-format-diff.py helps formatting for Google style. Such as the location of the opening braces.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/Tools#code-formatting

**Comment by elharo** [comment]
we're using apache maven style here, not google style

**Comment by ian-lavallee** [comment]
PR is very different than when created at this point so closing and opening a fresh PR

**Pull request closed without merging**